packages/romania/romania.svg

```diff
@@ -0,0 +1,174 @@
+     name="Botoșani"
+     id="bt" />
+  <path
+     d="m 384.73418,355.97362 -4.49,-0.7 -0.83,-1.85 0.11,-3.02 -0.53,0.32 -1.58,-0.26 -1.58,0.16 -2.74,1.59 -0.31,-0.25 -0.36,0.19 0.11,1.45 -0.52,-0.08 -1,0.78 -0.16,2.21 2.34,1.37 0.21,1.28 -3.37,0.16 -0.25,1.1 1.02,2.05 1.29,0 0.22,0.75 0.77,0.16 0.4,-0.46 1.23,1.4 0.35,-0.27 0.67,1.24 3.03,0.88 0.15,0.59 0.39,0 0.02,-0.67 0.65,-0.05 -0.11,0.45 0.55,0.57 0.62,-0.11 0.8,1.84 1.81,-0.67 -1.56,-2.96 1.23,-2.31 0.62,0.3 1.29,-0.32 0.64,-0.86 0.43,0.32 1.4,-2.77 -4.33,-1.42 0.56,-1.27 z"
+     name="Bucharest"
+     id="b" />
```

Comment:
Short names are OK, but I think 1 letter is too short 😄 
Could you use full names or ISO 3166-2 format for all the `id` please?